apps/backend/src/services/polling_service.py

```
@@ -68,14 +78,15 @@ async def stop_polling(self) -> None:
-                    await task
-                except asyncio.CancelledError:
-                    pass
+        # Cancel all polling tasks for all devices
+        for device_id, tasks in self.polling_tasks.items():
+            for task_type, task in tasks.items():
+                if not task.done():
+                    task.cancel()
+                    try:
+                        await task
+                    except asyncio.CancelledError:
+                        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(asyncio.CancelledError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(asyncio.CancelledError)`

(SIM105)